### ISSUE TITLE
do not go down to subspecies level 

### ISSUE THREAD

**Issue by alvarosaurus** [MfN-Berlin/AnimalAudiogramDatabase#10]
Reported by e-mail: MD Sa 23.01, 08:04, MD Sa 23.01, 03:55

276-274: please combine the two Species for the last two audiograsm that I send on great cormorant and this one – it is actually the same animal and we should not go down to subspecies level

174-177: change species to „Finless porpoise“ do not go down to subspecies level

**Comment by alvarosaurus** [comment]
Currently, we have subspecies: 
* Phoca vitulina vitulina
* Trichechus manatus latirostris
* Neophocaena asiaeorientalis asiaeorientalis
* Phalacrocorax carbo sinensis

**Comment by aot29** [comment]
Please [review pull request](https://github.com/MfN-Berlin/AnimalAudiogramDatabase_API/pull/1) @cboelling @miDaehne

**Comment by aot29** [comment]
> Reported by e-mail: MD Sa 23.01, 08:04, MD Sa 23.01, 03:55
> 
> 276-274: please combine the two Species for the last two audiograsm that I send on great cormorant and this one – it is actually the same animal and we should not go down to subspecies level
> 
> 174-177: change species to „Finless porpoise“ do not go down to subspecies level

These IDs appear to be not current anymore. The subspecies are listed in the filter pulldown "by species".

**Comment by cboelling** [comment]
For the time being subspecies should not be offered in the search interface. I think in the current database there are no valid datasets where the texon of the test animal is determined on the subspecies level.

**Comment by alvarosaurus** [comment]
Yes that's right, there should be no animals determined at subspecies level in the database. However, `Browse->Filter->By species` shows 2 entries at subspecies level. So there's a problem either with the data in the taxon table (not with he scheme, but with the data) or there's a problem with the query. I don't have access to the database so I can't check. Maybe somebody could send me a database dump by email?

**Comment by cboelling** [comment]
@miDaehne could you send Alvaro either a database dump or the access information for the database?
